tests/unit_tests/__init__.py

```diff
@@ -0,0 +1,10 @@
+import importlib
+import os
+import sys
+
+sys.path.append(os.path.join(os.path.dirname(__file__), '..', '..', '..'))
+
+import DOVE.src._utils as dutils
```

Comment:
I think we should talk about this pattern over a quick meeting. At first glance, I see adding these lines for the unit testing modules but having to add DOVE to the path for all the src files seems unnecessary since we have relative path access.